app/mails/management/commands/sendmail.py

```diff
@@ -16,12 +16,7 @@
 
 class Command(BaseCommand):
     def handle(self, *args, **kwargs):
-        try:
-            smtp = smtplib.SMTP(settings.EMAIL_HOST)
-            smtp.starttls()
-            smtp.login(settings.EMAIL_HOST_USER, settings.EMAIL_HOST_PASSWORD)
-        except:
-            return
+        smtp = connect_to_smtp()
```

Comment:
Same here - need to return if `smtp` fails.

... On the other hand, I'm not even sure we need this. We never use the `smtp` connection, and use Django's infrastructure for sending email. I think you can remove all this SMTP stuff here completely. No idea why it's even here :)

Reply:
ok, deleted the smtp part :)